apps/gp2-server/test/data-providers/news.data-provider.test.ts

```diff
@@ -154,6 +154,71 @@ describe('News data provider', () => {
         );
       });
     });
+
+    describe('Type Filter', () => {
+      test('Should query data properly when only News Type is selected', async () => {
+        contentfulGraphqlClientMock.request.mockResolvedValueOnce(
+          getContentfulNewsGraphqlResponse(),
+        );
+
+        const result = await newsDataProvider.fetch({
+          filter: { type: ['news'] },
+        });
+
+        expect(result).toEqual(getListNewsDataObject());
+        expect(contentfulGraphqlClientMock.request).toHaveBeenCalledWith(
+          expect.anything(),
```

Comment:
You could possibly replace expect.anything() in this spec with gp2Contentful.FETCH_NEWS -